src/alinea/adel/adel_dynamic.py

```diff
@@ -41,10 +41,10 @@ def add_metamer(self, g, plant=1, axe='MS'):
             shape_key = self.leaves[0].get_leaf_key(lctype, lcindex, age=None)
 
         internode_properties = {'ntop': ntop, 'length': 0, 'visible_length': 0,
-                                'senesced_length': 0, 'diameter': m['Ed'][0],
+                                'senesced_length': 0, 'diameter': m['Ed'][0] / 100,
```

Comment:
I don't think this is a good idea, as these values are provided by user at instantiation; Probably better to add declaration of dim/scene units in AdelWheat (see #6 ). I will revert this, and propose a new solution for #6